### PR TITLE
Added admin stats page

### PR DESCRIPTION
**Pull request by yugensoft**
Let me know if this covers what you needed

### CODE REVIEW

**Comment by Xon** [comment]
Can you inject the redis stats onto the home page above the Logged activity section? The styling will need changing to fit.

The redis info accessible via version is fairly hard to read, I think the css needs looking at;
image

The slave info doesn't open in an overlay either (OverlayTrigger is an XF1 class name)

File: upload/src/addons/SV/RedisCache/_output/templates/admin/SV_Redis_info.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<div class="section" id="RedisInfo">
+    <div class="secondaryContent statsList">
```
Comment (Xon) [inline upload/src/addons/SV/RedisCache/_output/templates/admin/SV_Redis_info.html]: This css needs porting to XF2 equivalents

File: upload/src/addons/SV/RedisCache/_output/templates/admin/SV_Redis_info_Wrapper.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<xf:title>{{ phrase('redis_info') }}</xf:title>
+
+<div class="splash">
```
Comment (Xon) [inline upload/src/addons/SV/RedisCache/_output/templates/admin/SV_Redis_info_Wrapper.html]: This wrapper class was used in XF2 to make the overlap popup work compared to the injected inline version.

**Review by Xon — CHANGES_REQUESTED** [review]

**Comment by yugensoft** [comment]
Alright I believe that covers it

**Comment by Xon** [comment]
LGTM

**Pull request merged**